### PR TITLE
Fix UI hiding while seeking or changing volume

### PR DESCRIPTION
**Pull request by vzablock**
Fixed UI hiding while seeking or changing volume.
Holding and dragging the sliders to change values won't make the UI hide anymore.

### CODE REVIEW

**Comment by vzablock** [comment]
@stonko1994 the `stopPropagation` would disable the `mousemove` event for showing and hiding UI that was bound in `uicontainer`.

File: src/ts/components/seekbar.ts [hunk 1]
```diff
@@ -689,10 +693,6 @@ export class SeekBar extends Component<SeekBarConfig> {
       e.preventDefault();
 
       if (seeking) {
-        // During a seek (when mouse is down or touch move active), we need to stop propagation to avoid
-        // the VR viewport reacting to the moves.
-        e.stopPropagation();
```
Comment (protyposis) [inline src/ts/components/seekbar.ts]: Why don't you make it conditional here, like above?

File: src/ts/components/seekbar.ts [hunk 2]
```diff
@@ -689,10 +693,6 @@ export class SeekBar extends Component<SeekBarConfig> {
...
-        // Because the stopped propagation inhibits the event on the document, we need to call it from here
```
Comment (protyposis) [inline src/ts/components/seekbar.ts]: This comment indicates that the next call was only present because of the propagation stop. Are you sure it makes sense to keep it?

File: src/ts/components/uicontainer.ts [hunk 3]
```diff
@@ -158,7 +158,7 @@ export class UIContainer extends Container<UIContainerConfig> {
       },
     }, {
       name: 'focusin',
-      handler: (e) => {
+      handler: () => {
```
Comment (protyposis) [inline src/ts/components/uicontainer.ts]: This change is completely unrelated to this PR.

Comment (vzablock) [reply src/ts/components/uicontainer.ts]: It is I agree but the argument is unused in the method (a bit of code cleaning doesn't hurt in the long run)

File: src/ts/components/seekbar.ts (hunk 2 above)
Comment (vzablock) [reply src/ts/components/seekbar.ts]: The method `mouseTouchMoveHandler` has the `e.stopPropagation();` call. Doesn't make sense to have the method being called 2 times. Same can be discussed regarding `e.preventDefault()`

File: src/ts/components/seekbar.ts (hunk 1 above)
Comment (vzablock) [reply src/ts/components/seekbar.ts]: We have the `mouseTouchHandler` being called which has the conditional in it

File: src/ts/components/uicontainer.ts (hunk 3 above)
Comment (protyposis) [reply src/ts/components/uicontainer.ts]: Code cleaning, when if unrelated to the PR, should be done in a separate PR.
